marklogic-data-hub-central/ui-custom/src/components/DataTableValue/DataTableValue.tsx

```diff
@@ -99,9 +99,7 @@ const DataTableValue: React.FC<Props> = (props) => {
                             if (d[props?.config?.value]) {
                                 value = d[props.config.value];
                             } else {
-                                if (Array.isArray(d)) {
```

Comment:
@wooldridge This check is necessary because when the entity doesn't have the configured property, the "d" parameter has an object. The application fails because react doesn't allow you to display an object. I added the array check because in some examples we have the property as an array of strings like ["value 1"," value 2"].

Reply:
If you have another case, we can add others checks

Reply:
Two tests fail on the current branch:

DEBUG_PRINT_LIMIT=10000 npm t DataTableValue.test
<img width="713" alt="Screen Shot 2022-05-16 at 1 07 38 PM" src="https://user-images.githubusercontent.com/477757/168673899-fdcc0293-5fd4-41d9-984b-9645709ec948.png">

The following fixes it and addresses your comment, I think, I will check in for your review:
```
let value;
                            if (d[props?.config?.value]) {
                                value = d[props.config.value];
                            } else {
                                if (Array.isArray(d) || !_.isObject(d)) {
                                    value = d;
                                }
                            }
```

Reply:
@wooldridge  it looks great, we can go with this